### ISSUE TITLE
Dynamic inClass dependant on outClass

### ISSUE THREAD

**Issue by drnen101** [blivesta/animsition#84]
I have a previous button and a next button that loads different pages. I would like to use fade-out-left, fade-out-right as the transition, but more specifically if you click NEXT i want inClass: 'fade-in-right', & outClass: 'fade-out-left', but if you click the previous button i would like to REVERSE it. It looks like i can define data-animsition-out-class directly on the link but im not sure how to change the inClass on the loading page. So i think i might need some sort of listener somewhere to catch this- any suggestions?

**Comment by ActiveScottShaw** [comment]
Yes, we are also after something similar, where it would be great to define different 'in-classes' per link clicked similar to how you can do it with the 'out-classes'.

Could this be achieved with it as is I wonder .... there are some 'methods' and 'events' .... but no real explanation/documentation as to what each does and how to use it correctly.
